pkg/recipes/types.go

```diff
@@ -169,31 +172,7 @@ func GetGitURL(templatePath string) (*url.URL, error) {
 	return url, nil
 }
 
-// GetEnvAppResourceNames returns the application, environment and resource names.
-func GetEnvAppResourceNames(resourceMetadata *ResourceMetadata) (string, string, string, error) {
-	app, err := resources.ParseResource(resourceMetadata.ApplicationID)
-	if err != nil {
-		return "", "", "", err
-	}
-
-	env, err := resources.ParseResource(resourceMetadata.EnvironmentID)
-	if err != nil {
-		return "", "", "", err
-	}
-
-	resource, err := resources.ParseResource(resourceMetadata.ResourceID)
-	if err != nil {
-		return "", "", "", err
-	}
+func hasURLScheme(url string) bool {
 
-	return env.Name(), app.Name(), resource.Name(), nil
-}
-
-// GetURLPrefix returns the url prefix to be added to the template path before adding it to the .gitconfig and terraform config.
-func GetURLPrefix(resourceRecipe *ResourceMetadata) (string, error) {
-	env, app, resource, err := GetEnvAppResourceNames(resourceRecipe)
-	if err != nil {
-		return "", err
-	}
-	return fmt.Sprintf("https://%s-%s-%s-", env, app, resource), nil
+	return len(strings.Split(url, "://")) > 1
```

Comment:
I was wondering if there's a specific reason for using a function here. Would inlining the expression directly in `GetGitURL` affect readability or ability to unit test?

Reply:
i was thinking this could be used in the future, i agree with inlining it and have a func when needed. Updated it